File: 001/trapezoid area/trapezoid_area.dart
trapezoid_area(a, b, h)
{
  // if we need to typecate the result to int
  // return ((a + b) * h / 2).toInt();
  // or return (a + b) * h ~/ 2;
  return (a + b) * h / 2;
}


main() {
  print(trapezoid_area(2, 4, 2));
}

File: 001/trapezoid area/trapezoid_area.cpp
#include <iostream>
using namespace std;

float trapezoid_area(float a, float b, float h)
{
    return (a + b) * h / 2;
}

int main()
{
    cout << "Area: " << trapezoid_area(5, 10, 4) << endl;
    return 0;
}